### PR TITLE
Add reference to pypi distribution in the README

### PR DESCRIPTION
**Pull request by sbesson**
Similar to what has been done in `omero-marshal`, this PR adds a link to the `pypi` distribution of `omego`.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -41,5 +45,3 @@ Copyright
 2013-2014, The Open Microscopy Environment
```
Comment (mtbc) [inline README.rst]: No later? (Not this PR's issue.)

**Comment by mtbc** [comment]
Staged at https://github.com/sbesson/omego/tree/pypi.

**Comment by aleksandra-tarkowska** [comment]
Do you know about https://testpypi.python.org/pypi ?

**Comment by sbesson** [comment]
Haven't investigated it thoroughly although this might be a solution for deploying development distributions which has been mentioned at several occasions. What's the reason for mentioning it in the context of this PR?

**Comment by mtbc** [comment]
Good to merge.

**Comment by joshmoore** [comment]
Very nice.

**Pull request merged**